bootstrap/docs/manual_search.py

```
@@ -0,0 +1,121 @@
+            connection.execute(create_sql)
+
+        doc_count = 0
+        for doc in _iter_docs(root):
+            doc_count += 1
+            id_tokens = doc_id_tokens(doc.id)
+            for locale, localization in doc.localizations.items():
+                table = FTS_TABLES[locale]
+                title = normalize_search_text(localization.title)
+                body = normalize_search_text(strip_markdown(localization.body_markdown))
+                connection.execute(
+                    f"INSERT INTO {table}(doc_id, title, body) VALUES (?, ?, ?)",
```

Ruff (0.16.0)

[error] Possible SQL injection vector through string-based query construction

(S608)